### ISSUE TITLE
I found that the bounding box loss function uses GIOU loss and L1 loss, and the hyperparameter settings are the same as TransT. Have you tried DIOU or CIOU? Do you have any experience with hyperparameter settings? thanks a lot！

### ISSUE THREAD

**Issue by wildwolff** [MCG-NJU/MixFormer#2]

**Comment by yutaocui** [comment]
We haven't try other loss in order to take a relatively fair comparison with STARK/TransT. 
For hyperparameter settings, I usually train different-setting models for a few epochs.
